sys/auto_init/auto_init.c

```diff
@@ -28,18 +28,18 @@
 
 void auto_init(void)
 {
-   if (IS_USED(MODULE_AUTO_INIT_ZTIMER)) {
+    if (IS_USED(MODULE_AUTO_INIT_ZTIMER)) {
         LOG_DEBUG("Auto init ztimer.\n");
         void ztimer_init(void);
         ztimer_init();
     }
     if (IS_USED(MODULE_AUTO_INIT_XTIMER) &&
-            !IS_USED(MODULE_ZTIMER_XTIMER_COMPAT)) {
+        !IS_USED(MODULE_ZTIMER_XTIMER_COMPAT)) {
```

Comment:
`&&` at ENDL
```
RIOT$ rg -t c -e '&&$' |wc -l
685
```
i also had a look into the context and it commonly indented like the pr (no extra indent for the second line)


I would like this:    /* but it seems less common */
`&&` at Begin of line  
```
RIOT$ rg -t c -e '^\s*&&'|wc -l
80
```

Reply:
i couldn't find anything about this in https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md or the referenced linux convention.
none of our code-checks complain

Reply:
Since it is not in the Coding conventions for RIOT (nor the Linux conventions), I personally see no reason to go from good code to slightly less readable code. 

But it's already ACK'ed and merged, so no point in discussing it further :)